pkg/binary/envoy/run.go

```diff
@@ -102,11 +112,7 @@ func (r *Runtime) AppendArgs(args []string) {
 
 func (r *Runtime) waitForExit(cancel context.CancelFunc) {
```

Comment:
actually do we need this? looks like only used in one place

Reply:
noticed also!